fast-markdown-mcp/src/fast_markdown_mcp/server.py

```
@@ -189,6 +210,205 @@ async def search_files(self, query: str) -> str:
         except Exception as e:
             logger.error(f"Error searching files: {e}")
             return f"Error searching files: {str(e)}"
+            
+    async def smart_section_search(self, query: str, max_results: int = 10,
```

Ruff (0.8.2)

`smart_section_search` is too complex (22 > 10)

(C901)